SasquatchMac/SasquatchMacObjC/AppDelegate.m

```diff
@@ -12,8 +12,8 @@ @interface AppDelegate ()
 @implementation AppDelegate
 
 - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
-
-  // Insert code here to initialize your application
+  [MSAnalytics setEnabled:[ServiceStateStore AnalyticsState]];
```

Comment:
The SDK will read the value from storage in each service to set enable/disable for initial state. You can remove setEnabled call in AppDelegate.